tasks/configure.yml

```diff
@@ -138,6 +138,7 @@
       --username="{{ archivematica_src_configure_am_user }}"
       --password="{{ archivematica_src_configure_am_password }}"
       --email="{{ archivematica_src_configure_am_email }}"
+      --site-url="{{ archivematica_src_configure_am_site_url | default ('127.0.0.1') }}"
```

Comment:
since `archivematica_src_configure_am_site_url` is defined in `defaults/main.yml` the default('127.0.0.1') filter may not be needed?